clients/src/main/java/org/apache/kafka/clients/consumer/internals/Coordinator.java

```diff
@@ -217,21 +217,6 @@ public void ensureCoordinatorKnown() {
         }
     }
 
-    /**
```

Comment:
We may have lost valuable information about how async commits can block on the coordinator when this comment was deleted. Not critical, especially since this is internal code, but you might want to restore that comment by moving it to `commitOffsetsAsync`.